app/queries/control_rate_cohort_query.rb

```diff
@@ -80,7 +80,7 @@ def overall_controlled_bps_per_facility
   def overall_patients
     @overall_patients ||=
       Patient
-        .for_reports(exclude_ltfu_as_of: Date.today.end_of_month)
+        .for_reports(exclude_ltfu_as_of: Period.month(Date.today).end)
```

Comment:
Mind adding a little factory method to Period to make this easier?

```ruby
Period.current.end
```

It could default to month period, because we use that in 95% of cases...we could add an optional arg version for quarters if we want:

```ruby
Period.current(:quarter).end
```

Reply:
Oh, and this should use `end_time` to be more explicit here (its aliased right now but `end` should really go away at some point...).